core/src/main/java/com/novoda/merlin/receiver/ConnectivityChangeEvent.java

```diff
@@ -29,4 +33,43 @@ public String getReason() {
     public NetworkStatus asNetworkStatus() {
         return isConnected() ? NetworkStatus.newAvailableInstance() : NetworkStatus.newUnavailableInstance();
     }
+
+    @Override
+    public String toString() {
+        return "ConnectivityChangeEvent{" +
```

Comment:
Not sure if project convention here or not, but we usually put the `+`s on the beginning of the new line instead.